### ISSUE TITLE
using 'Skip.If(true, comment)' wipes out any previous XUnitOutput.WriteLine

### ISSUE THREAD

**Issue by brewmanz** [AArnott/Xunit.SkippableFact#16]
I dump out debugging info via XUnitOutput.WriteLine, and then I find a condition that requires the Skip feature, but the XUnit test 'Output' has lost debugging info, and only Skip comment shows. 
It would be better if both the XUnitOutput.WriteLine as well as the Skip comment would show in the test 'Output' (and also the Skip comment be visible in the XUnit side window like say "Message: The collection was expected to contain a single element, but it was empty." rather than having to view 'Output').

**Comment by AArnott** [comment]
I agree, but at the moment I'm not sure that's something I can fix at this level. That might be an issue with xunit itself.
